code/graphics/gropenglbmpman.cpp

```diff
@@ -40,7 +40,7 @@ int get_num_mipmap_levels(int w, int h)
 	int size, levels = 0;
 
 	// make sure we can and should generate mipmaps before trying to use them
```

Comment:
I think this comment should be adjusted to explain what is actually happening, after all, checking for EXT_FRAMEBUFFER_OBJECT does not seem to have anything to do with mipmapping